packages/runtime/container-runtime-definitions/src/containerRuntime.ts

```diff
@@ -26,6 +27,11 @@ import {
 	IProvideFluidDataStoreRegistry,
 } from "@fluidframework/runtime-definitions";
 
+export interface IContainerRuntimeWithResolveHandle_Deprecated extends IContainerRuntime {
```

Comment:
Mark this as deprecated as well for emphasis (this is what we did for the bindToContext one).